File: System/LogicServer/Entities/Language.cs
using System;

namespace Entities;

public class Language : IIdentifiable<string>{
    public string Code { get; set; } = string.Empty;
    public string Name { get; set; } = string.Empty;
    public string Id 
    { 
        get => Code; 
        set => Code = value; 
    }
}

File: Appendices/3.1 Source Code/System/DataServer/DataServer/src/main/java/via/sep3/dataserver/data/Language.java
package via.sep3.dataserver.data;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "language", schema = "learn_db")
public class Language
{
  @Id
  private String code;

  private String name;

  public String getCode()
  {
    return code;
  }

  public void setCode(String code)
  {
    this.code = code;
  }

  public String getName()
  {
    return name;
  }

  public void setName(String name)
  {
    this.name = name;
  }
}
